elvis.config

```diff
@@ -27,7 +27,8 @@
                  %% Allow usage of ?assertMatch and similar macros, since after expanding they
                  %% result in use of variables like __X and __V that wouldn't be permitted by
                  %% default regex.
-                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z_]*)$"}}
+                 {elvis_style, variable_naming_convention, #{regex => "^(_?[A-Z][0-9a-zA-Z_]*)$"}},
+                 {elvis_style, macro_names, disable}
```

Comment:
Is it official? I mean, do we really stop checking for illegal use of macro names?